Loadouts/aaf_f2000_combat.hpp

```diff
@@ -0,0 +1,222 @@
+// Author: Raeth
+// Description: AAF: F2000 - Gray/Green
+
+#include "undef.hpp" // Reset defines
+
+// Rifle
+#define RIFLE "arifle_Mk20_F"
+#define RIFLE_MAG "30Rnd_556x45_Stanag:8","30Rnd_556x45_Stanag_Tracer_Yellow:2"
+#define RIFLE_ATTACHMENTS "optic_ACO_grn","acc_pointer_IR"
+#define AAR_ATTACHMENTS RIFLE_ATTACHMENTS
+#define ALT_OPTICS "rhsusf_acc_compm4","rhsusf_acc_eotech_xps3","CUP_optic_CompM2_Black","CUP_optic_TrijiconRx01_black","BWA3_optic_RSAS","RKSL_optic_RMR_MS19"
+// GL Rifle
+#define GLRIFLE "arifle_Mk20_GL_F"
+#define GLRIFLE_MAG RIFLE_MAG
+#define GLRIFLE_MAG_SMOKE "1Rnd_Smoke_Grenade_shell:2","1Rnd_Smoke_Grenade_shell:2"
+#define GLRIFLE_MAG_HE "1Rnd_HE_Grenade_shell:5"
+#define GLRIFLE_MAG_FLARE "UGL_FlareYellow_F:4"
+// Carbine
+#define CARBINE "arifle_Mk20C_F"
+#define CARBINE_MAG RIFLE_MAG
+// AR
+#define AR "LMG_Mk200_F" // #Stoner
+#define AR_MAG "200Rnd_65x39_cased_Box:2"
+// AT
+#define AT "potato_nlaw_dumb"
+// MMG
+#define MMG "rhs_weap_m240B" //That moment when Bi never made a AAF MMG :thinking:
```

Comment:
I think the 240 fits more thematically with their other weapons, they don't really have any "future" weapons so it would be out of place.